yarn-project/deploy_npm.sh

```diff
@@ -8,52 +8,52 @@ cd project/src/yarn-project
 echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
 
 function deploy_package() {
-    REPOSITORY=$1
-    VERSION=$(extract_tag_version $REPOSITORY false)
-
-    # If the commit tag itself has a dist-tag (e.g. v2.1.0-testnet.123), extract the dist-tag.
-    TAG=$(echo "$VERSION" | grep -oP ".*-\K(.*)(?=\.\d+)" || true)
-    TAG_ARG=""
-    if [ -n "$TAG" ]; then
-        TAG_ARG="--tag $TAG"
-    fi
-
-    readonly PUBLISHED_VERSION=$(npm show . version ${TAG_ARG:-} 2> /dev/null)
-    readonly HIGHER_VERSION=$(npx semver ${VERSION} ${PUBLISHED_VERSION} | tail -1)
-
-    # If there is already a published package equal to given version, assume this is a re-run of a deploy, and early out.
-    if [ "$VERSION" == "$PUBLISHED_VERSION" ]; then
-        echo "Tagged version $VERSION is equal to published version $PUBLISHED_VERSION. Skipping publish."
-        exit 0
-    fi
-
-    # If the published version is > the given version, something's gone wrong.
-    if [ "$VERSION" != "$HIGHER_VERSION" ]; then
-        echo "Tagged version $VERSION is lower than published version $PUBLISHED_VERSION."
-        exit 1
-    fi
-
-    # Update the package version in package.json.
-    TMP=$(mktemp)
-    jq --arg v $VERSION '.version = $v' package.json > $TMP && mv $TMP package.json
-
-    if [ -z "$STANDALONE" ]; then
+  REPOSITORY=$1
```

Comment:
Has this file changed other than whitespace?

Reply:
indeed no, will remove from PR